packer/scylla.json

```diff
@@ -63,7 +63,12 @@
       "ssh_timeout": "5m",
       "ssh_username": "{{user `ssh_username`}}",
       "ssh_clear_authorized_keys": true,
-      "subnet_id": "{{user `subnet_id`}}",
+      "subnet_filter": {
+          "filters": {
+              "tag:Name": "ami-subnet*"
```

Comment:
Maybe `image-build-subnet`? since it's related to our building process

Reply:
ok, changed